pkg/utils/convert_test.go

```diff
@@ -0,0 +1,115 @@
+package utils
+
+import (
+	"fmt"
+	"testing"
+	"time"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestConvertToFloat64(t *testing.T) {
```

Comment:
Since this tests other functions besides `ConvertToFloat64()`, I would rename it to something like `TestConvertToNumericTypes()`